demos/demo-minimal-js/index.js

```diff
@@ -46,18 +46,21 @@ function enableButtons(ids) {
 const cti = new CallingExtensions({
   debugMode: true,
   eventHandlers: {
-    onReady: data => {
+    onReady: ({ engagementId, portalId } = {}) => {
```

Comment:
We could add the `userId` as well if the integrator needs it to get the `ownerId`